### PR TITLE
update go interface

### PR DESCRIPTION
**Pull request by ForLisiteng**

### CODE REVIEW

File: sdk/go/createkey.go [hunk 1]
```diff
@@ -1,94 +1,91 @@
...
 
 import (
-    hmac "crypto/hmac"
-    sha256 "crypto/sha256"
-    "encoding/base64"
-    "encoding/json"
-    "fmt"
-    ioutil "io/ioutil"
-    "time"
-    "strconv"
```
Comment (syan10) [inline sdk/go/createkey.go:11]: asymetric_encrypt not update?

File: sdk/go/createkey.go [hunk 2]
```diff
@@ -1,94 +1,91 @@
...
-
-    "github.com/iancoleman/orderedmap"
+	hmac "crypto/hmac"
+	sha256 "crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
```
Comment (syan10) [inline sdk/go/createkey.go:8]: do we have unitest for go interfaces?

File: sdk/go/createkey.go [hunk 3]
```diff
@@ -1,94 +1,91 @@
...
+	} else {
+		return "", fmt.Errorf("Please input keyspec.")
+	}
+	if origin != "" {
+		payload.Set("origin", origin)
+	} else {
+		return "", fmt.Errorf("Please input origin.")
+	}
+
+	if keyusage != "" {
```
[the remark this answers is not in the document]
Comment (ForLisiteng) [reply sdk/go/createkey.go:38]: fix

File: sdk/go/createkey.go (hunk 2 above)
Comment (ForLisiteng) [reply sdk/go/createkey.go:8]: unit testing will be added on 289

File: sdk/go/createkey.go (hunk 1 above)
Comment (ForLisiteng) [reply sdk/go/createkey.go:11]: update on 289

Comment (syan10) [reply sdk/go/createkey.go:11]: this pr will closed?

**Pull request closed without merging**